src/mpl/include/mpl_base.h

```diff
@@ -56,6 +56,16 @@
 #define MPL_STATIC_INLINE_PREFIX ATTRIBUTE((always_inline)) static inline
 #define MPL_STATIC_INLINE_SUFFIX ATTRIBUTE((always_inline))
 
+/* Ideally this would use a better check to look like something above, but
+ * there isn't a built in way to detect whether the fallthrough attribute
+ * exists in autoconf. For now, the easiest way is to look for GCC 7.x or
+ * higher */
+#if __GNUC__ >= 7
+#define MPL_FALLTHROUGH ATTRIBUTE((fallthrough))
+#else
+#define MPL_FALLTHROUGH
+#endif
```

Comment:
This should be a configure check, not just a check for `__GNUC__`.

Reply:
Ah. I finally found the check for this in the autoconf archive. Removed this bandaid.